.envrc

```diff
@@ -0,0 +1,14 @@
+if command -v nix &>/dev/null; then
+    # Load .envrc from certora dir if exists
+    [ -f ../.envrc ] && source_env ..
+    # Use nix if it's available
+    use flake
+fi
```

Comment:
so to be clear, what's the behavior if `nix` is unavailable?

Reply:
So if the nix is not available `direnv` will only create a virtual environment using the stdlib `venv` package and activate the virtual environment.